### ISSUE TITLE
itextpdf 5.5.9 is removed. Can't build.

### ISSUE THREAD

**Issue by GaryH4** [Audiveris/audiveris#305]
When first run `./gradlew build`, get this feedback:
```
* What went wrong:
Could not resolve all files for configuration ':compileClasspath'.
> Could not download itextpdf.jar (com.itextpdf:itextpdf:5.5.9)
   > Could not get resource 'https://jcenter.bintray.com/com/itextpdf/itextpdf/5.5.9/itextpdf-5.5.9.jar'.
      > Read timed out
```

**Comment by GaryH4** [comment]
Modefied file `gradlew.build`, changed version of itextpdf to the latest avaliable version `5.5.6`.
Seems fine now, but don't know if any problem will occur.

**Comment by maximumspatium** [comment]
I just built Audiveris on Windows from scratch by deleting the local Gradle cache. That caused Gradle to reload all dependencies.
itextpdf 5.5.9 is still there. Everything works as expected.

If it breaks with the "time out" error, just try again. File removal should cause another kind of error ("file not found" or "file not available" or similar).

itextpdf is used for exporting recognized music to PDF...

**Comment by GaryH4** [comment]
I see.
It was a time out error. And I tried open the url which gradle was going to fetch, only 5.5.6 and earlier versions are shown.

Later I found my device(Nvidia jetson Nano) is on arm architecture, so I just installed the x86 version on my Windows laptop.

Now I have some other issue on import a pdf file, which leads to mis-recognition.
Definitions of pdf files after import in Audiveris is much lower. I'm looking for somewhere adjust it.
